src/anomalib/utils/callbacks/__init__.py

```diff
@@ -16,24 +16,20 @@
 
 from anomalib.deploy import ExportMode
 
-from .cdf_normalization import CdfNormalizationCallback
 from .graph import GraphLogger
 from .metrics_configuration import MetricsConfigurationCallback
-from .min_max_normalization import MinMaxNormalizationCallback
 from .model_loader import LoadModelCallback
 from .post_processing_configuration import PostProcessingConfigurationCallback
 from .tiler_configuration import TilerConfigurationCallback
 from .timer import TimerCallback
 from .visualizer import ImageVisualizerCallback, MetricVisualizerCallback
 
 __all__ = [
-    "CdfNormalizationCallback",
```

Comment:
I just thought that since we assign normalization callbacks in the trainer, I should make them part of the trainer's callbacks. But maybe I can keep them here and add a note to the docstring

Reply:
Just had a second thought. I think it would still be more organized if we keep the callbacks in one place. We might need to re-organize the relative imports to not be able to import these callbacks easily

https://github.com/openvinotoolkit/anomalib/blob/ed1d9fa5c7050a6edf79febd256d27b261a441ae/src/anomalib/utils/callbacks/__init__.py#L31

Thoughts?